### DISCUSSION TITLE
Navigator [Bug]

### DISCUSSION THREAD

**Discussion by shzyb** [webstudio-is/webstudio-community#34] [Q&A]
Whenever I copy and paste any thing inside a box, it doesn't stay select and automatically goes to the copied element and if I paste again it is pasted inside the copied element.

Can someone please fix this. 

<img width="277" alt="Screenshot 2023-06-13 at 9 47 56 PM" src="https://github.com/webstudio-is/webstudio/assets/102948337/0db2540d-b3e1-4385-ac5c-201bc13dcf0e">

**Comment by kof** [comment]
Do I understand correctly that when you paste an instance, the pasted instance gets selected, but you want the original instance to stay selected?

**Comment by kof** [comment]
Yep I got it, you are correct, this is annoying!

**Comment by TrySound** [comment]
Fixed in https://github.com/webstudio-is/webstudio-builder/pull/1782. Soon will be released.
